actionmailer/lib/action_mailer/railtie.rb

```diff
@@ -59,6 +59,14 @@ class Railtie < Rails::Railtie # :nodoc:
       end
     end
 
+    initializer "action_mailer.set_autoload_paths" do |app|
+      options = app.config.action_mailer
+
+      if options.show_previews && options.preview_path
+        ActiveSupport::Dependencies.autoload_paths << options.preview_path
+      end
+    end
```

Comment:
I really like that setting autoload paths is split from the other initializer. 👍